### PR TITLE
Slides: Add Open Force Field talk slides

### PR DESCRIPTION
**Pull request by j-wags**
If you are creating this PR in order to submit a draft of your paper,
see http://procbuild.scipy.org/ for logs generated by the build
process.

See the [project readme](https://github.com/scipy-conference/scipy_proceedings#build-your-paper)
for more information.

*Editor*:  

*Reviewers*:

### CODE REVIEW

File: presentations/slides/openforcefield/info.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "title": "Open Force Field: next-generation force fields with open data, open software, and open science
+",
```
Comment (jakirkham) [inline presentations/slides/openforcefield/info.json]:
```suggestion
    "title": "Open Force Field: next-generation force fields with open data, open software, and open science",
```

Comment (j-wags) [reply presentations/slides/openforcefield/info.json]: Thanks!

**Review by deniederhut — APPROVED** [review]
Absolutely did a double take at this talk title. 

tenor-1986314839

**Comment by j-wags** [comment]
Thanks so much for running a great conference!

**Comment by deniederhut** [comment]
🙏

**Pull request merged**